src/OrchardCore/OrchardCore.Data.Abstractions/Documents/IDocumentFileStore.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Data.Documents
+{
+    /// <summary>
+    /// A Singleton service used to persist documents to file storage.
+    /// </summary>
+    public interface IDocumentFileStore
```

Comment:
I don't think we need another interface for this. The whole logic could be contained in `FileDocumentStore`. It's not as much extensible, but anyone can create another implementation if they don't like how it's saved in files (where and with which format).